desc/compat.py

```diff
@@ -261,3 +261,72 @@ def rescale(
     eq.surface = eq.get_surface_at(rho=1)
 
     return eq
+
+
+def rotate_zeta(eq, angle, copy=False):
+    """Rotate the equilibrium about the toroidal direction.
+
+    Parameters
+    ----------
+    eq : Equilibrium
+        Equilibrium to rotate.
+    angle : float
+        Angle to rotate the equilibrium in radians. The actual physical rotation
+        is by angle radians. Any rotation that is not a multiple of pi/NFP will
+        break the symmetry of a stellarator symmetric equilibrium.
+    copy : bool, optional
+        Whether to update the existing equilibrium or make a copy (Default).
+
+    Returns
+    -------
+    eq_rotated : Equilibrium
+        Equilibrium rotated about the toroidal direction
+    """
+    eq_rotated = eq.copy() if copy else eq
+    # We will apply the rotation in NFP domain
+    angle = angle * eq.NFP
+    # Check if the angle is a multiple of pi/NFP
+    kpi = np.isclose(angle % np.pi, 0, 1e-8, 1e-8) or np.isclose(
```

Comment:
one more question, what if `angle` is negative? should this not have an abs value on the angle?

Reply:
I am pretty sure `mode` operator `%` only returns positive numbers.

Reply:
add neg angle test